usage/rest-api/src/main/java/org/apache/brooklyn/rest/domain/CatalogEntitySummary.java

```diff
@@ -46,6 +56,7 @@ public CatalogEntitySummary(
             @JsonProperty("planYaml") String planYaml,
             @JsonProperty("description") String description,
             @JsonProperty("iconUrl") String iconUrl,
+            @JsonProperty("tags") Set<Object> tags,
```

Comment:
True. My first implementation didn't use tags to store TOSCA metadata, that's why I modified the `CatalogEntitySummary`. I'll modify that.